lms/static/sass/course/_student-notes.scss

```diff
@@ -172,10 +181,6 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
               background: transparent;
             }
 
-            .note-comment-p {
```

Comment:
NOTE: And the designer said, "let the students see what they hath commented in its entirety".